src/app/shared/components/xterm-terminal/terminal-toolbar/component.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
+
+@Component({
+  selector: 'km-terminal-toolbar',
+  templateUrl: './template.html',
+  styleUrls: ['./style.scss'],
+})
+export class TerminalToolBarComponent implements OnInit {
+  @Input() showCloseIcon = true;
+  @Input() showOpenInNewTabIcon = true;
+
+  @Output() closeTerminal = new EventEmitter<boolean>();
+  @Output() openInNewTab = new EventEmitter<boolean>();
+
+  constructor() {}
+
+  ngOnInit(): void {}
```

Comment:
Remove empty `ngOnInit` and `constructor` here as well

Reply:
IMHO, constructor is part of class and it is default generated that must be there but yes empty lifecycle hook should be removed. thoughts?

Reply:
i guess code coverage is also not covering which causes pipeline for coverage to fail also

Reply:
fixed